examples/benchmarks/LightGBM/multi_freq_handler.py

```diff
@@ -0,0 +1,123 @@
+#  Copyright (c) Microsoft Corporation.
+#  Licensed under the MIT License.
+
+import warnings
+from pathlib import Path
+from typing import Union
+import pandas as pd
+
+from qlib.data import D
+from qlib.data.dataset.loader import QlibDataLoader
+from qlib.contrib.data.handler import DataHandlerLP, _DEFAULT_LEARN_PROCESSORS, check_transform_proc
+
+
+class MultiFreqLoader(QlibDataLoader):
+    def load_group_df(
+        self,
+        instruments,
+        exprs: list,
+        names: list,
+        start_time: Union[str, pd.Timestamp] = None,
+        end_time: Union[str, pd.Timestamp] = None,
+        gp_name: str = None,
+    ) -> pd.DataFrame:
+        if instruments is None:
+            warnings.warn("`instruments` is not set, will load all stocks")
+            instruments = "all"
+        if isinstance(instruments, str):
+            instruments = D.instruments(instruments, filter_pipe=self.filter_pipe)
+        elif self.filter_pipe is not None:
+            warnings.warn("`filter_pipe` is not None, but it will not be used with `instruments` as list")
+
+        if gp_name == "feature":
+            # freq == day
+            _exps = list(filter(lambda x: not x.startswith("Avg"), exprs))
```

Comment:
Why is this class necessary?

Reply:
1. feature_day,  feature_15min,   label
2. resample 
3. concat

loader :  [feature_day,  feature_15min,   label]  vs [feature, label]